common/src/main/scala/com/gu/media/youtube/package.scala

```diff
@@ -23,26 +21,80 @@ package object youtube {
       YouTubeVideoCategory(category.getId.toInt, category.getSnippet.getTitle)
     }
   }
+  
+  case class YouTubeChannel(id: String, title: String)
 
   object YouTubeChannel {
-    implicit val reads: Reads[YouTubeChannel] = (
-      (__ \ "title").read[String] ~
-        (__ \ "logo").read[String].map(URI.create) ~
-        (__ \ "id").read[String]
-      )(YouTubeChannel.apply _)
-
-    implicit val writes: Writes[YouTubeChannel] = (
-      (__ \ "title").write[String] ~
-        (__ \ "logo").write[String].contramap((_: URI).toString) ~
-        (__ \ "id").write[String]
-      )(unlift(YouTubeChannel.unapply))
+    implicit val reads: Reads[YouTubeChannel] = Json.reads[YouTubeChannel]
+    implicit val writes: Writes[YouTubeChannel] = Json.writes[YouTubeChannel]
 
     def build(channel: Channel): YouTubeChannel = {
-
       YouTubeChannel(
-        title = channel.getSnippet().getTitle(),
-        logo = URI.create(channel.getSnippet().getThumbnails().getDefault().getUrl()),
-        id = channel.getId
+        id = channel.getId,
+        title = channel.getSnippet.getTitle
+      )
+    }
+  }
+
+  case class YouTubeVideo (
+     id: String,
+     title: String,
+     duration: Long,
+     publishedAt: DateTime,
+     privacyStatus: String,
+     tags: Seq[String],
+     contentBundleTags: Seq[String],
+     channel: YouTubeChannel
+  )
+
+  object YouTubeVideo {
```

Comment:
Maybe we could call this something else? The youtube videos trait already has a `getVideo` method that returns objects of type `Video`so this feels a bit confusing.